miner/worker.go

```diff
@@ -1587,6 +1590,8 @@ func (w *worker) commit(uncles []*types.Header, interval func(), update bool, st
 	if err != nil {
 		return err
 	}
+	l2CommitTxsPerBlockGauge.Update(int64(len(w.current.txs)))
```

Comment:
Is the one you're adding the same purpose of my PR (https://github.com/scroll-tech/go-ethereum/pull/713/files)? If so I'll close my PR.